### PR TITLE
Linking to demo and websites

### PR DESCRIPTION
**Pull request by ginotoro**
Adding links to:
* Avatar Stack Demo
* Examples page on the website for both root and the Avatar Stack's readme files

### CODE REVIEW

**Review by artismarti — APPROVED** [review]

File: src/components/AvatarStack/README.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ _Link to the Avatar Stack example and demo._
 
 ## Resources
 
-_Links to relevant pages. See www.ably.com/topic/avatar-stack. We link to a lot of resources there. Method links._
+- Read further about what is possible with an [avatar stack](https://ably.com/examples/avatar-stack?utm_source=ably-labs&utm_medium=github&utm_campaign=avatar-stack).
```
Comment (artismarti) [inline src/components/AvatarStack/README.md:25]: Can we use 'what is possible' in just one of these sentences & reword the other one @ginotoro

**Review by artismarti — COMMENTED** [review]
just a small copy change request.

File: src/components/AvatarStack/README.md (hunk 1 above)
Comment (ginotoro) [reply src/components/AvatarStack/README.md]: Updated

File: src/components/AvatarStack/README.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ _Link to the Avatar Stack example and demo._
 
 ## Resources
 
-_Links to relevant pages. See www.ably.com/topic/avatar-stack. We link to a lot of resources there. Method links._
+- Read further more about building an [avatar stack](https://ably.com/examples/avatar-stack?utm_source=ably-labs&utm_medium=github&utm_campaign=avatar-stack).
```
Comment (artismarti) [inline src/components/AvatarStack/README.md:25]:
```suggestion
- Learn more about how to build your own [avatar stack](https://ably.com/examples/avatar-stack?utm_source=ably-labs&utm_medium=github&utm_campaign=avatar-stack).
```

File: src/components/AvatarStack/README.md (hunk 1 above)
Comment (artismarti) [reply src/components/AvatarStack/README.md]: Made another suggestion! :)

**Pull request merged**